packages/mui-material/src/Step/Step.test.js

```diff
@@ -121,4 +121,45 @@ describe('<Step />', () => {
       expect(stepLabels[1]).to.have.class(stepLabelClasses.disabled);
     });
   });
+
+  describe('reflects props in classNames', () => {
+    it('renders with "active" className when active', () => {
+      const { container } = render(
+        <Step active>
+          <StepLabel>Step 1</StepLabel>
+        </Step>,
+      );
+
+      const root = container.querySelector(`.${classes.root}`);
+      const label = container.querySelector(`.${stepLabelClasses.root}`);
```

Comment:
This works but let's use `getByText` here:
```suggestion
      const label = getByText('Step 1');
```

This is the recommended way in React testing library because of the reasons mentioned [here]( https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-container-to-query-for-elements).